ingestion/functions/parsing/cuba/cuba.py

```diff
@@ -0,0 +1,180 @@
+import os
+import sys
+from datetime import datetime
+import json
+import pycountry
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    Dates are listed in YYYY/mm/dd format
+    """
+    date = datetime.strptime(raw_date, "%Y/%m/%d")
+    return date.strftime("%m/%d/%Y")
```

Comment:
please make it "%m/%d/%YZ" so that the date filtering works as well (it expects the timezone Z at the end)